### ISSUE TITLE
Too slow in Firefox

### ISSUE THREAD

**Issue by suhr** [mlajtos/L1#3]
With `::Self` it barely responses to typing.

Browser: Firefox 62.0b13

**Comment by mlajtos** [comment]
I tried Firefox and it is **really** slow. It hangs for seconds! I haven’t investigated why is it so slow.

I put a notice on the main GitHub page that Chrome is preferred.

Resolution for now: Use Chrome

**Comment by mlajtos** [comment]
This is strange. Firefox is unusuable not only during rendering but also after the rendering is done, which is beyond my understanding. (It is constantly relayouting the page.) Also, there is white rectangle where supposed to be content. `¯\_(ツ)_/¯`

I also tried FF52 (guessing that Quantum is the culprit) and it is also slow and unusable, but at least the rendering is okay.

There are few things that might be investigated:
1. using font without ligatures
1. not using `mix-blend-mode: multiply`
1. fix the scrolling container for the visualization (`will-change: transform`)

I also noticed that `image-rendering: pixelated` does not work on FF, which is essential for the visualization.

---

**Conclusion:**
This is clearly problem of FF. (Chrome, Safari, Brave, Opera are without a problem.) I can't force Mozilla to fix their browser. So, Firefox is bad, use Chrome.
